### DISCUSSION TITLE
In some areas,Some resources load slowly

### DISCUSSION THREAD

**Discussion by wlfcss** [requarks/wiki#3414] [Help / Questions]
**Question**
I am a developer from China,I found that when opening the wiki, some resources load slowly or fail directly~
E.g : https://use.fontawesome.com/releases/v5.10.0/css/all.css

**Host Info (please complete the following information):**
I hope to manually change the loading address of these resources, but I don’t know how to modify it easily, I hope I can get some help

**Comment by hell1ano** [comment]
Having found the same issue on my offline installation.. did you found any solution or where you could set this manually?

**Comment by hell1ano** [comment]
@wlfcss I solved it manually in my installation, see https://github.com/Requarks/wiki/issues/3117#issuecomment-787054867
